qa/tasks/cephfs/test_client_limits.py

```diff
@@ -38,15 +38,16 @@ def _test_client_pin(self, use_subdir, open_files):
         :param use_subdir: whether to put test files in a subdir or use root
         """
 
-        cache_size = open_files/2
+        cache_memory_limit = open_files/2 * 2000
```

Comment:
I think you can just set this to a really low number, like "1M". Have you tried running it?

Reply:
Done